lib/core/connection_factory/http.rb

```diff
@@ -5,7 +5,9 @@
 module Core
   module ConnectionFactory
     class Http
-      def self.build(url, timeout: 5, open_timeout: 5, retries: 2)
+      TIMEOUT = ENV.fetch('FRONTEND_HTTP_REQUEST_TIMEOUT').to_i
```

Comment:
I'm okay with this implementation and not adding defensive code at this level. We should be able to trust that the script project will the correct env var since it goes through the PR thumbing process